src/views/DocumentationTopic.vue

```diff
@@ -319,13 +350,22 @@ export default {
         this.toggleMobileSidenav();
       }
     },
+    openQuickNavigationModal() {
```

Comment:
I feel like all of the logic related to `showQuickNavigationModal`, like listeners and all, can be inside the `<QuickNavigationModal>`, so its contained and does not spill into the main `DocumentationTopic.vue` component

Reply:
`openQuickNavigationModal` and `showQuickNavigationModal` has to be accessed outside the `QuickNavigationModal` component to open the modal, like in the top navbar and the keydown event listener. Having everything contained in a single component would be ideal, but I think that is not possible with the current implementation.

Reply:
Oh right right....